src/config_dialog.c

```diff
@@ -102,9 +102,9 @@ static void syntax(char **words, unsigned int words_len) {
 static void read_only_cmd(char **words, unsigned int words_len) {
```

Comment:
By the way I am not sure why read_only mode can be changed, since this is detected at the file system level the file couldn't be saved anyway

Reply:
I would like to do it like Emacs. I think a `can_write` boolean would be good to know if we can turn `read_only` off,